.github/workflows/azure-site-extension.yml

```diff
@@ -0,0 +1,78 @@
+name: Azure Site Extension
+
+on:
+  workflow_dispatch:
+
+env:
+  SPEC_FILE_TEMPLATE: 'NewRelic.Azure.WebSites.Extension.NodeAgent.nuspec'
+
+jobs:
+  setup_tools:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup dotnet '6.0.x'
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '6.0.x'
+      - name: Display dotnet version
+        run: dotnet --version
+      - name: Display NuGet version
+        run: nuget ?
+
+  create_extension_bundle:
+    runs-on: windows-latest
+    needs:
+      - setup_tools
+
+    strategy:
+      matrix:
+        node-version: ['18.x', '20.x'] # Node 22 isn't yet available for Windows apps in Azure
```

Comment:
can you remove `.x`? this ends up putting that in the package name which is very confusing

Reply:
also im not certain we need a site extension per node version. we have 2 native libraries: `@newrelic/native-metricss` and `@contrast/fn-inspect` and they provide prebuilts for every supported node version

Reply:
turns out our native metrics module works across versions but the `@contrast/fn-inspect` does not.  it only has prebuilt for win32-x64 not ia32 which we need apparently

Reply:
@bizob2828 I don't know if we need ia32--looking at the VM options in Azure, they allow x86-64 and ARM64. I can add a matrix to publish for those two arch options.

Reply:
Azure does let users provision VMs for ARM, but not for Windows as the OS. As site extensions aren't (yet) supported for Linux, we don't have to publish for ARM. 

In the case of 32-vs-64...the default for deploying into Azure is x86-64. x86-32 can be provisioned in Azure, but it's not an easy process: https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows/support-32-bit-operating-systems-virtual-machines. As such, 32-bit seems like an edge case.

Reply:
I would completely ignore x86 in this day and age. Very few OS vendors are continuing to directly support it.

Reply:
Then we need to triage why the prebuilts aren't working for @contrast/fn-inspect. I was at loss when I looked Friday

Reply:
I'm not sure if I was clear. I'm saying all of the major operating systems are only shipping 64-bit builds when the target platform is Intel. That is, Windows x64 or Linux amd64. We included x86 for Windows in `@newrelic/native-metrics` because people are able to deploy on systems that manage themselves and possibly still utilize Windows x86.

Reply:
After testing, we've determined that building for x86 doesn't give us any advantage: x86 applications will still fall back to function name for CLM, due to the lack of an x86 build of `fn-inspect`. In our tests, we've found an x64-built site extension to instrument equally well for x86 and x64 in every respect other than CLM.